app/views/sessions/show.html.slim

```diff
@@ -0,0 +1,14 @@
+h1.mb-3
+  | Your profile info
+p.lead
+  strong
+    | First Name:
```

Comment:
It would be better to use `'` instead of `|` because we need a space between `First Name:` and name. `'` add space at the and of the string line